railties/test/application/test_runner_test.rb

```diff
@@ -446,7 +446,7 @@ def test_pass_TEST_env_on_rake_test
       create_test_file :models, 'account'
       create_test_file :models, 'post', pass: false
 
-      output =  Dir.chdir(app_path) { `bin/rake test TEST=test/models/post_test.rb` }
+      output =  Dir.chdir(app_path) { `bin/rails test TEST=test/models/post_test.rb` }
```

Comment:
@senny is this supposed to be supported only on rake, or this should be fixed on `/bin/rails` as well?
https://travis-ci.org/rails/rails/jobs/104328919#L927